src/celutil/debug.h

```diff
@@ -20,6 +20,10 @@
 #undef DPRINTF // OSX has DPRINTF
 #endif // DPRINTF
 
+#define PRINT_LEVEL_ERROR       0
```

Comment:
Verbosity levels are usually ordered from Error to Debug, ie Debug is more verbise than normal info and normal info is mire verbose than error. In more sophisticated systems I saw following levels:
error
warning
info
verbose
debug